<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TexturePropertiesView</class>
 <widget class="QDockWidget" name="TexturePropertiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>375</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#QDockWidget {
   margin: 10px;
   border: 10px solid black;
padding: 20px;
}</string>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>   Texture Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget{
border: 2px solid rgb(135, 177, 255);
background: rgb(194, 203, 255);
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../shaderlab.qrc">
           <normaloff>:/ico/img/Add-icon.png</normaloff>:/ico/img/Add-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="loadButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../shaderlab.qrc">
           <normaloff>:/ico/img/loadtexture.png</normaloff>:/ico/img/loadtexture.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../shaderlab.qrc">
           <normaloff>:/ico/img/removetexture.png</normaloff>:/ico/img/removetexture.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">#widget_2{
border: 2px solid rgb(135, 177, 255);
background: rgb(194, 203, 255);
border-radius: 10px;

}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="textureList"/>
       </item>
       <item>
        <widget class="QLabel" name="imageContainer">
         <property name="styleSheet">
          <string notr="true">#imageContainer{
border: 1px solid black;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../shaderlab.qrc">:/ico/noImage</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../shaderlab.qrc"/>
 </resources>
 <connections/>
</ui>
